src/google/protobuf/compiler/objectivec/objectivec_helpers.cc

```diff
@@ -209,10 +209,14 @@ const char* const kReservedWordList[] = {
 hash_set<string> kReservedWords =
     MakeWordsMap(kReservedWordList, GOOGLE_ARRAYSIZE(kReservedWordList));
 
-string SanitizeNameForObjC(const string& input, const string& extension) {
+string SanitizeNameForObjC(const string& input,
```

Comment:
A Bool result would seem to impl success/failure of the call.  The call always returns the string, the bool ptr is optional, and allows callers to see if the suffix was added also.